Lib/fontTools/ttLib/__main__.py

```diff
@@ -108,10 +108,16 @@ def main(args=None):
             collection = TTCollection(f, lazy=lazy)
             fonts.extend(collection.fonts)
 
-    if lazy is False:
-        for font in fonts:
-            for table in tables if "*" not in tables else font.keys():
-                font[table]  # Decompiles
+    if tables is None:
+        if lazy is True:
+            tables = []
+        else:
+            tables = ["*"]
```

Comment:
I think lazy=None by default, so if you want to avoid decompile all tables by default, when neither `-t '*' or `--no-lazy` options are passed, then you could do:

```suggestion
        elif lazy is False:
            tables = ["*"]
```

Reply:
What to set in the case of lazy None? I need to set *something*.

Reply:
Empty list?

Reply:
Yes, that's what my latest change does. Thanks.